src/main/java/com/hazelcast/cloud/ClientWithSsl.java

```diff
@@ -78,6 +80,24 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static void fetchCitiesThroughJdbc(ClientConfig config, Properties tlsProperties) {
+        String jdbcUrl = String.format("jdbc:hazelcast://%s/?discoveryToken=%s&cloudUrl=%s&sslEnabled=true",
+                config.getClusterName(),
+                config.getNetworkConfig().getCloudConfig().getDiscoveryToken(),
+                config.getProperties().getProperty("hazelcast.client.cloud.url"));
+        try (Connection connection = DriverManager.getConnection(jdbcUrl, tlsProperties);
+             Statement statement = connection.createStatement();
+             ResultSet rs = statement.executeQuery("SELECT this FROM cities")) {
+            System.out.println("--Fetching the results through JDBC interface:");
```

Comment:
```suggestion
            Statement statement = connection.createStatement();
            ResultSet rs = statement.executeQuery("SELECT this FROM cities")) {
            System.out.println("--Fetching the results through JDBC interface:");
```

Reply:
Statement and ResultSet are indented differently because they are part of try-with-resources block, while println belongs to the try block.

Reply:
Oh OK, I have missed that 👍